pkg/eks/api.go

```diff
@@ -63,6 +63,9 @@ func (p ProviderServices) STS() stsiface.STSAPI { return p.sts }
 // Region returns provider-level region setting
 func (p ProviderServices) Region() string { return p.spec.Region }
 
+// Version returns provider-level version setting
+func (p ProviderServices) Version() string { return p.spec.Version }
```

Comment:
Hm, I would put this under cluster actually, provider is supposed to be very high-level.